### PR TITLE
Azure pipelines

### PR DESCRIPTION
**Pull request by nicksterx**
Completes #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 [![Apache 2.0 license](https://img.shields.io/github/license/sandialabs/gulliver?logo=apache)](https://github.com/sandialabs/Gulliver/blob/master/LICENSE)
 [![.NetStandard 1.3](https://img.shields.io/badge/targets-.NETStandard%201.3-5C2D91?logo=.net)](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
 [![Join the chat at https://gitter.im/sandialabs/Gulliver](https://badges.gitter.im/sandialabs/Gulliver.svg)](https://gitter.im/sandialabs/Gulliver?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](https://dev.azure.com/sandianationallabs/Gulliver/_apis/build/status/sandialabs.Gulliver?branchName=master)](https://dev.azure.com/sandianationallabs/Gulliver/_build/latest?definitionId=1&branchName=master)
```
Comment (rheone) [inline README.md]: how about code coverage
 Azure DevOps coverage

```markdown
![Azure DevOps coverage](https://img.shields.io/azure-devops/coverage/sandianationallabs/Gulliver/1)
```
and tests
Azure DevOps tests (compact)

```markdown
![Azure DevOps tests (compact)](https://img.shields.io/azure-devops/tests/sandianationallabs/Gulliver/1?compact_message)
```
and

Comment (nicksterx) [reply README.md]: yes! Let me add those and turn on the code coverage results, which are missing.

Comment (rheone) [reply README.md]: @nicksterx, how is versioning handled for releases? Tagging?
Does it auto release on nugget for us?

Comment (rheone) [reply README.md]: > yes! Let me add those and turn on the code coverage results, which are missing.

Cool, will review in a bit. I need to climb onto the roof and decommission the swamp cooler. If I don't approve within an hour call an ambulance / mortition.

Comment (nicksterx) [reply README.md]: Haha I will!

Comment (nicksterx) [reply README.md]: Now it's ready!

File: README.md [hunk 2]
```diff
@@ -6,6 +6,9 @@
 [![.NetStandard 1.3](https://img.shields.io/badge/targets-.NETStandard%201.3-5C2D91?logo=.net)](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
 [![Join the chat at https://gitter.im/sandialabs/Gulliver](https://badges.gitter.im/sandialabs/Gulliver.svg)](https://gitter.im/sandialabs/Gulliver?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://dev.azure.com/sandianationallabs/Gulliver/_apis/build/status/sandialabs.Gulliver?branchName=master)](https://dev.azure.com/sandianationallabs/Gulliver/_build/latest?definitionId=1&branchName=master)
+![Azure DevOps coverage](https://img.shields.io/azure-devops/coverage/sandianationallabs/Gulliver/1)
+![Azure DevOps tests (compact)](https://img.shields.io/azure-devops/tests/sandianationallabs/Gulliver/1?compact_message)
```
Comment (rheone) [inline README.md]: How about a hard break in here keeping all the build stuff on its own line? It looks cluttered as is

image

Comment (nicksterx) [reply README.md]: image

**Pull request merged**